src/libs/actions/Report.js

```diff
@@ -1109,6 +1119,84 @@ function updateLastReadActionID(reportID, sequenceNumber, manuallyMarked = false
     });
 }
 
+/**
+ * Gets all the report actions and updates the last read message
+ *
+ * @param {Number} reportID
+ */
+function openReport(reportID) {
+    const sequenceNumber = reportMaxSequenceNumbers[reportID];
+    API.write('OpenReport',
+        {
+            reportID,
+            sequenceNumber,
+        },
+        {
+            optimisticData: [{
+                onyxMethod: 'merge',
+                key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+                value: {
+                    lastVisitedTimestamp: Date.now(),
```

Comment:
Sure, just, 2 observations, I copied this from `setLocalLastRead` so we've been setting `Date.now()`. 
And this value never reaches the server, but I guess we should make it the same as what the server sends back since the server will be setting this in openReport.

Reply:
Yep I'm aware that the existing logic is wrong. And still think it's a good opportunity to fix it and not continue to do something that is wrong.

Reply:
Sorry, I need to mention that I realized I was totally wrong about this. Did some digging and basically...

- report action timestamps are seconds since epoch
- this lastVisitedTimestamp is milliseconds since epoch